scripts/no-changes-in-high-risk-files.sh

```diff
@@ -3,14 +3,19 @@ set -euo pipefail
 
 # To update the sha run the command in ACTUAL and copy the result in EXPECTED.
 
-ACTUAL=$(shasum -a 256 ./Sources/Sentry/SentryNSURLSessionTaskSearch.m ./Sources/Sentry/SentryNetworkTracker.m)
+ACTUAL=$(shasum -a 256 ./Sources/Sentry/SentryNSURLSessionTaskSearch.m ./Sources/Sentry/SentryNetworkTracker.m ./Sources/Sentry/SentryUIViewControllerSwizzling.m ./Sources/Sentry/SentryNSDataSwizzling.m ./Sources/Sentry/SentrySubClassFinder.m)
 EXPECTED="819d5ca5e3db2ac23c859b14c149b7f0754d3ae88bea1dba92c18f49a81da0e1  ./Sources/Sentry/SentryNSURLSessionTaskSearch.m
-58d5414b4f0a4c821b20fc1a16f88bda3116401e905b7bc1d18af828be75e431  ./Sources/Sentry/SentryNetworkTracker.m"
+58d5414b4f0a4c821b20fc1a16f88bda3116401e905b7bc1d18af828be75e431  ./Sources/Sentry/SentryNetworkTracker.m
+52cb473dcc8d13c0d4f6cd1429c3fc6e8588521660b714f4a2edb4eaf1401e9f  ./Sources/Sentry/SentryUIViewControllerSwizzling.m
+e95e62ec7363984f20c78643bb7d992a41a740f97e1befb71525ac34caf88b37  ./Sources/Sentry/SentryNSDataSwizzling.m
+9ad05dd8dd29788cba994736fdcd3bbde59a94e32612640d11f4f9c38ad6610e  ./Sources/Sentry/SentrySubClassFinder.m"
 
 if [ "$ACTUAL" = "$EXPECTED" ]; then
     echo "No changes in high risk files."
     exit 0
 else
-    echo "Changes in high risk files. If your changes are intended please update the sha in ./scripts/no-changes-in-high-risk-files.sh to $ACTUAL."
+    echo "Changes in high risk files. Test your changes by using `make test-alamofire` and `make test-homekit` to ensure the changes are safe to run on third party projects."
```

Comment:
`l`: I think we should change this to the following, as testing the changes with `make test-alamofire` and  `make test-homekit` might not make sense for every change in these files.
```suggestion
    echo "Changes in high risk files. You might want to test your changes by using `make test-alamofire` and `make test-homekit` to ensure the changes are safe to run on third party projects."
```